### ISSUE TITLE
Add "clear log" and "clear row" options during a trade dump

### ISSUE THREAD

**Issue by MonkeyJamboree** [Cu3PO42/KeySAVe#185]
To help facilitate trade (TEA) dumps when something goes wrong, possibly include an easy-to-access "clear" button along with the copy, file, and export buttons, or allow removal of one or more rows in the log of what's been dumped so far.

Use case: get to the end of a box while dumping and you only have 28 or 29 of the expected 30.

Use case 2: dumping of multiple trainers' trades in one session without having to reconnect.

**Comment by Cu3PO42** [comment]
Wouldn't a button clearing a single box also be helpful for the first use case you mentioned?

**Comment by MonkeyJamboree** [comment]
Yes, exactly. I suppose this would technically be two features. I wasn't sure how to write it as such though. One feature being an individual line x button or clear button, another being a whole-log clear button.
